lib/Sema/TypeCheckDeclObjC.cpp

```diff
@@ -1748,7 +1748,7 @@ static ObjCSelector inferObjCName(ValueDecl *decl) {
                                 diag::objc_override_method_selector_mismatch,
                                 *attr->getName(), overriddenSelector)
                       .limitBehavior(behavior));
-              fixDeclarationObjCName(diag, decl, attr->getName(),
+              fixDeclarationObjCName(diag, decl, attr->getName().value(),
```

Comment:
Reviewers: Note that `value()` here is safe because one of the conditions above ensures that `attr->hasName()` is `true`. That's something to check for when you're examining this PR.